galileo_sdk/galileo_cli/lzs.py

```diff
@@ -0,0 +1,87 @@
+import click
+import pandas
+
+from galileo_sdk import GalileoSdk
+from halo import Halo
+
+
+def lzs_cli(main, galileo: GalileoSdk):
+    @main.group()
+    def lzs():
+        """
+        Get information about your Landing Zones.
+        """
+        pass
+
+    @lzs.command()
+    @click.option(
+        "--lz_ids", 
+        multiple=True, 
+        type=str, 
+        help="Filter by Landing Zone id."
+    )
+    @click.option(
+        "--userid",
+        required=False,
+        type=str,
+        help="Filter by page number."
+    )
+    @click.option(
+        "--page", 
+        type=int, 
+        help="Filter by page number."
+    )
+    @click.option(
+        "--items", 
+        type=int, 
+        help="Filter by number of items in the page.",
+    )
+    @click.option(
+        '-a', 
+        '--all', 
+        is_flag=True
+    )
+    def ls(lz_ids, userid, page=0, items=10, all=False):
+        """
+        List all Landing Zones in your Galileo account.
+        """
+        
+        spinner = Halo("Retrieving lzs", spinner="dot")
+        spinner.start()
+
+        self = galileo.profiles.self()
+        userids = []
+        if not all:
+            userids.append(self.userid)
+
+        lzs = galileo.lz.list_lz(
+           lz_ids=list(lz_ids), userids=list(userids), page=page, items=items,
+        )
+
+        if len(lzs) == 0:
+            spinner.stop()
+            click.echo("No Landing Zones found.")
+            return
+
+
+        lzs = [lz.__dict__ for lz in lzs]
+
+        lzs_df = pandas.json_normalize(lzs)
+        lzs_df = lzs_df[
+            [
+                "name",
+                "arch",
+                "status",
+                "userid",
+                "cpu_count",
+                "gpu_count",
+                "memory_amount",
+            ]
+        ]
+
+        spinner.stop()
+        if items:
```

Comment:
I'm not sure this truth check is required since it has a default parameter value of 10